### PR TITLE
Minor improvements to Ledger unittests

### PR DESCRIPTION
**Pull request by Geod24**
All to make it easier to switch to a default of 20 for ValidatorCycle.

### CODE REVIEW

File: source/agora/node/Ledger.d [hunk 1]
```diff
@@ -1572,24 +1572,23 @@ unittest
     import agora.consensus.data.genesis.Coinnet : CoinGenesis = GenesisBlock;
```
Comment (linked0) [inline source/agora/node/Ledger.d:1572]: Can't we remove `CoinGenesis`, using `GenesisBlock` instead?

**Review by linked0 — APPROVED** [review]
LGTM, and one question.

**Review by hewison-chris — APPROVED** [review]
LGTM

File: source/agora/node/Ledger.d (hunk 1 above)
Comment (Geod24) [reply source/agora/node/Ledger.d:1572]: No because it is still used (we need two Genesis block to have a mismatch).

**Review by kchulj — APPROVED** [review]
LGTM

**Pull request merged**